tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -173,8 +173,31 @@ def test_bucket_delete(
                 *["OC", {"interface": "OC", "backingstore_dict": {"gcp": [(1, None)]}}],
                 marks=[tier1],
             ),
+            pytest.param(
+                *[
+                    "OC",
+                    {"interface": "OC", "backingstore_dict": {"ibmcos": [(1, None)]}},
+                ],
+                marks=[tier1],
+            ),
+            pytest.param(
+                *[
+                    "CLI",
+                    {"interface": "OC", "backingstore_dict": {"ibmcos": [(1, None)]}},
+                ],
+                marks=[tier1],
+            ),
+        ],
+        ids=[
+            "S3",
+            "CLI",
+            "OC",
+            "OC-AWS",
+            "OC-AZURE",
+            "OC-GCP",
+            "OC-IBMCOS",
+            "CLI-IBMCOS",
```

Comment:
Does it require to have some secret for IBM COS added in cluster?

Reply:
Hi @petr-balogh 
 this requires a section added to auth.yaml. Here is a sample:
{'AUTH':{'IBMCOS': {
'SECRET_PREFIX': 'IBM_COS',
'DATA_PREFIX': 'IBM_COS',
'ENDPOINT': 'https://s3.us-east.cloud-object-storage.appdomain.cloud',
'REGION': 'us-east',
'IBM_COS_ACCESS_KEY_ID': 'xxxxx',
'IBM_COS_SECRET_ACCESS_KEY': 'xxxxx',
'AWS_SDK_LOAD_CONFIG': 1
},
.......
}
}

Reply:
@gitsridhar for IBM Cloud COS data we already have some different structure in auth config.

See this file:
https://github.com/red-hat-storage/ocs-ci/blob/6ecc96ca640e96448a16e2271f5553a7a6248c12/docs/supported_platforms.md#configuration-files

It's used for whole deployment. Can we be consistent and use the same structure of data?

Also I would like to see this documented. Without this comment I will not know what structure it needs. It should be documented.

Also my question is as you are adding it to parametrized. What will happen if those data will not be set in the auth.yaml? Will the test fail for IBM COS?  Or will it be skipped?

Thanks

Reply:
@petr-balogh the auth config format was provided to us by Ben Eli and that format is required for our object storage to work. We will add a paragraph in ocs-ci documentation about this.
Regarding the second part, if IBM COS  options are not provided in auth.yaml, the tests will be skipped just like how it works for aws , gcp.

Reply:
Ok, if you were given this structure by @MeridianExplorer thank use it for now. But can you please open the issue for consolidating this with only one structure? As we already using one for deployment and here the other. Thanks